packages/pg-pubsub/__tests__/subs.test.ts

```diff
@@ -78,7 +93,8 @@ describe("Middleware defaults", () => {
       {},
       async (json, _req, res) => {
         expect(res.statusCode).toEqual(200);
-        expect(printIntrospectionResults(json)).toMatchSnapshot();
+        const schema = buildClientSchema(json.data);
```

Comment:
Let's add another assertion just in case
```suggestion
        expect(json.errors).toBeFalsy();
        const schema = buildClientSchema(json.data);
```